packages/console/src/pages/TenantSettings/TenantMembers/hooks.ts

```diff
@@ -1,38 +1,22 @@
 import { useContext, useMemo } from 'react';
 
 import { SubscriptionDataContext } from '@/contexts/SubscriptionDataProvider';
-import {
-  hasReachedSubscriptionQuotaLimit,
-  hasSurpassedSubscriptionQuotaLimit,
-} from '@/utils/quota';
 
 const useTenantMembersUsage = () => {
-  const { currentSubscriptionUsage, currentSubscriptionQuota } =
-    useContext(SubscriptionDataContext);
+  const {
+    currentSubscriptionUsage,
```

Comment:
This line seems can be removed since the usage is no longer used?

Reply:
This is used to get the `usage` variable, which is used with the `useTenantMembersUsage()` hook.